### PR TITLE
Second talk sketch

### PR DESCRIPTION
**Pull request by jimmyp**

### CODE REVIEW

File: MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrderPlaced.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace MarkAndJimsDealOfTheDay.PlacingOrders
+{
+    public class PotentialOrderPlaced : IEvent
+    {
+        public string ProductCode { get; }
+        public Guid Id { get; }
+        public int Quantity { get; set; }
+    }
```
Comment (hombredequeso) [inline MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrderPlaced.cs:10]: Should have:
public Guid CustomerId {get;}

and get set appropriately.

File: MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrder.cs [hunk 2]
```diff
@@ -1,46 +1,10 @@
...
-            var result = PotentialOrder.BuildOrder(customerId, productCode, quantity);
-
-            _uow.Repository.Save(result.Entity);
-            _uow.Bus.Publish(result.Event);
-            _uow.Commit();
-        }
-    }
-
     public class PotentialOrder : IEntity
     {
```
Comment (hombredequeso) [inline MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrder.cs:9]: Probably should have an orderId:

public Guid OrderId {get;}

then constructor is:
private PotentialOrder(Guid orderId, Guid customerId, string productCode, int quantity)
... etc

**Review by hombredequeso — COMMENTED** [review]
Looks good. Introduces a whole heap of concepts and is still easy to follow (I think :-) )

File: MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrder.cs (hunk 2 above)
Comment (jimmyp) [reply MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrder.cs:9]: 👍

File: MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrderPlaced.cs (hunk 1 above)
Comment (jimmyp) [reply MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrderPlaced.cs:10]: 👍

**Pull request merged**